src/App.css

```diff
@@ -59,3 +82,174 @@ body {
     transform: rotate(360deg);
   }
 }
+
+.action {
+  width: 4.063rem;
+  height: 1.125rem;
+  margin: 0 1.438rem 0 0;
+  opacity: 0.5;
+  font-size: 1.1rem;
+  padding: 0 0 0 1.2%;
+}
+
+.title {
+  width: 20.313rem;
+  height: 1.813rem;
+  margin: 0.488rem 20.188rem 0 0;
+  font-size: 1.375rem;
+  letter-spacing: -0.2px;
+  color: #121212;
+  padding: 0 0 0 1.2%;
+}
+
+.general {
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.gene {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.genero {
+  font-family: 'RobotoSlab', sans-serif;
+}
+
+.genex {
+  font-weight: bold;
+}
+
+.author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-size: 0.875rem;
+  color: #4386bf;
+  padding: 0 0 0 1.2%;
+}
+
+.font-3 {
+  font-weight: 300;
+}
+
+.comment {
+  width: 4.563rem;
+  height: 1.198rem;
+  margin: 1.188rem 0.938rem 0.125rem 0;
+  font-size: 0.999rem;
+  color: #4386bf;
+  background-color: #fff;
+  border: none;
+}
+
+.button {
+  cursor: pointer;
+}
+
+.span {
+  width: 0.1rem;
+  height: 1.5rem;
+  margin: 1rem 1.063rem 0 0.938rem;
+  border: solid 1px #e8e8e8;
+  background-color: #e8e8e8;
+}
+
+.progress-bar-c {
+  display: flex;
+  gap: 1.25rem;
+  width: 30%;
+}
+
+.progress-bar {
+  height: 70px;
+  width: 70px;
+}
+
+.progress-value-c {
+  display: flex;
+  flex-direction: column;
+  margin-top: 5px;
+}
+
+.progress-percentage {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 30px;
+  margin: 0;
+  color: #121212;
+  font-weight: 500;
+}
+
+.progress-completed {
+  font-family: 'Montserrat', sans-serif;
+  margin: 5px 0 0 0;
+  color: #121212;
+  opacity: 0.5;
+  font-size: 15px;
+  letter-spacing: normal;
+}
+
+.update-progg-c {
+  display: flex;
+  flex-direction: column;
+}
+
+.chapter-h2 {
+  width: 11.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-size: 0.813rem;
+  color: #121212;
+}
+
+.chapter-h3 {
+  width: 7.688rem;
+  height: 1.313rem;
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 1rem;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
```

Comment:
- [ ] Nice work implementing most of the changes requested by the former reviewer. I guess you forgot these as well. Kindly remove this declaration `font-family: 'RobotoSlab', sans-serif;` and use the `.genero` CSS property you created for on the element instead 👍